templates/docs/examples/patterns/breadcrumbs-deprecated.html

```diff
@@ -0,0 +1,15 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Breadcrumbs - deprecated{% endblock %}
```

Comment:
We usually title the examples as "Component / Something"

```suggestion
{% block title %}Breadcrumbs / Deprecated{% endblock %}
```